examples/opf/tools/sp_plotter.py

```diff
@@ -330,17 +330,16 @@ def testSPNew():
   
   
   # Setup a SP
-  sp = FDRCSpatial2.FDRCSpatial2(
```

Comment:
This and other examples still use the FDRCSpatial2 instead of new spatial pooler. So I had to convert them.